dm/tests/openapi/run.sh

```diff
@@ -209,25 +212,76 @@ function test_dump_task() {
 	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
 		"query-status $task_name" \
 		"\"stage\": \"Stopped\"" 1
+	openapi_task_check "check_task_stage_success" $task_name 1 "Stopped"
 
 	init_dump_data
 
 	# start dump task success
 	openapi_task_check "start_task_success" $task_name ""
-	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
-		"query-status $task_name" \
-		"\"stage\": \"Running\"" 1
```

Comment:
in some case, task will finished immediately so cannot get the `Running` stage. Only validate task finished or not